File: sandbox/ruby/src/ApiAppDeleteExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

begin
    Dropbox::Sign::ApiAppApi.new.api_app_delete(
        "0dd3b823a682527788c4e40cb7b6f7e9", # client_id
    )
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling ApiAppApi#api_app_delete: #{e}"
end

File: examples/ApiAppDeleteExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");
// $config->setAccessToken("YOUR_ACCESS_TOKEN");

try {
    (new Dropbox\Sign\Api\ApiAppApi(config: $config))->apiAppDelete(
        client_id: "0dd3b823a682527788c4e40cb7b6f7e9",
    );
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling ApiAppApi#apiAppDelete: {$e->getMessage()}";
}
